demo_nodes_cpp/CMakeLists.txt

```diff
@@ -41,7 +41,8 @@ function(add_demo_dependencies library)
     "rclcpp"
     "rclcpp_components"
     "rcutils"
-    "std_msgs")
+    "std_msgs"
+    "rcl_interfaces")
```

Comment:
Alphabetize, please (this should go after `rclcpp_components`).

Reply:
Fixed it